mesop/labs/chat.py

```diff
@@ -200,6 +202,8 @@ def submit():
         yield
     state.in_progress = False
     yield
+    me.focus_component(key=f"{len(state.output)}")
```

Comment:
I'm wondering why this needs to be in a separate yield block, i.e. can this be moved to above L204?

Reply:
I'm not sure if you're doing this because the focus command executes too early (e.g. before the component-to-be-focused had been rendered/updated). If that's the case, maybe in shell.ts, you can do some kind of trick like `setTimeout(() => { // focus...}, 0)`

(p.s. maybe scroll_into_view has the same issue)

Reply:
Ah good catch. Looks like that after yield was not needed.